<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1019</width>
    <height>657</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Music Player</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListView" name="directory_listview">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>661</width>
      <height>611</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>390</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="control_layout">
     <item>
      <widget class="QPushButton" name="previous_button">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Previous</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="play_pause_button">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Play</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="next_button">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Next</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>440</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="shuffle_repeat_layout">
     <item>
      <widget class="QPushButton" name="shuffle_button">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Shuffle</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="repeat_button">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Repeat</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QProgressBar" name="song_progressbar">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>490</y>
      <width>311</width>
      <height>23</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>311</width>
      <height>23</height>
     </size>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="song_length">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>520</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="current_progress">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>520</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>300</y>
      <width>311</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="song_information_layout">
     <item>
      <widget class="QLabel" name="artist_label">
       <property name="maximumSize">
        <size>
         <width>289</width>
         <height>36</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="title_label">
       <property name="maximumSize">
        <size>
         <width>289</width>
         <height>36</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="album_cover_label">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>10</y>
      <width>311</width>
      <height>271</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>311</width>
      <height>271</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1019</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_directory"/>
    <addaction name="actionEnable_dark_mode"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuPlayer">
    <property name="title">
     <string>Player</string>
    </property>
    <addaction name="actionShuffle"/>
    <addaction name="actionRepeat"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuPlayer"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_directory">
   <property name="text">
    <string>Open directory</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Q</string>
   </property>
  </action>
  <action name="actionShuffle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Shuffle</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="actionRepeat">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Repeat</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="actionEnable_dark_mode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Enable dark mode</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
